### PR TITLE
Support new attributes

### PR DESCRIPTION
**Pull request by jonathan-rvezy**
- Adding support for the store attribute on the field
- Adding support for reading the geo_distance_meters from the search result.

### CODE REVIEW

**Comment by runeanielsen** [comment]
Hi @jonathan-rvezy,

Can you please use the default indention level (4 spaces) that has been used across the project?

// Rune

**Comment by jonathan-rvezy** [comment]
> Hi @jonathan-rvezy,
> 
> Can you please use the default indention level (4 spaces) that has been used across the project?
> 
> // Rune

I changed it; it was the line endings that were the cause of the problem.

File: test/Typesense.Tests/TypesenseClientTests.cs [hunk 1]
```diff
@@ -1924,13 +1927,13 @@ public async Task Update_document_by_query()
...
         };
-        
+
         var expected = new FilterUpdateResponse(1);
         var response = await _client.UpdateDocuments("companies", document, "num_employees:>100");
 
         response.Should().BeEquivalentTo(expected);
     }
-    
+
```
Comment (runeanielsen) [inline test/Typesense.Tests/TypesenseClientTests.cs:1936]: Please remove this change.

File: test/Typesense.Tests/TypesenseClientTests.cs [hunk 2]
```diff
@@ -1924,13 +1927,13 @@ public async Task Update_document_by_query()
         {
             CompanyName = "Dom Bomb Dot Com",
         };
-        
+
```
Comment (runeanielsen) [inline test/Typesense.Tests/TypesenseClientTests.cs:1930]: Please remove this change.

File: src/Typesense/SearchResult.cs [hunk 3]
```diff
@@ -49,6 +49,25 @@ public record Highlight
...
+            return (double?)property.GetValue(this);
+        }
+        init
+        {
+            var property = GetType().GetProperty(propertyName) ?? throw new ArgumentException($"Property {propertyName} does not exist.");
+            property.SetValue(this, value);
+        }
+    }
+}
+
```
[the remark this answers is not in the document]
Comment (jonathan-rvezy) [reply src/Typesense/SearchResult.cs:70]: oh yeah good catch I originally was doing to use it that way but changed to a Dictionary instead.

**Comment by jonathan-rvezy** [comment]
@runeanielsen all requested changes have been made.

**Comment by runeanielsen** [comment]
@jonathan-rvezy looks good!, thanks for the contribution.

**Comment by jonathan-rvezy** [comment]
@runeanielsen will a new nuget package be released? My devops guys will be forever thankful so we don't need to load in our own version.

**Comment by runeanielsen** [comment]
@jonathan-rvezy I've now released a version 7.9.0 with the new changes.

**Pull request merged**